.github/workflows/ci.yml

```diff
@@ -60,7 +60,9 @@ jobs:
 
       - name: Install macOS dependencies
         if: contains(matrix.os, 'macos')
-        run:
+        run: |
+          brew update 
+          brew upgrade
```

Comment:
I think only `brew update` is required here.

Reply:
OK, I'll try that, thanks!